docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3376,6 +3376,25 @@ The same can be achieved by adding this rule:
 
 We recommend to use this kind of exceptions only if it is not possible to change the hiding rule itself. In other cases it is better to change the original rule, using domain restrictions.
 
+:::note CSS injection
+
+In AdGuard products which use **CoreLibs, version 1.18 and later**, element hiding rules may also be used to inject CSS, for example:
+
+```adblock
+example.org##body { background-color: green; }
+```
+
+this usage is discouraged in favor of using [CSS rules](#cosmetic-css-rules) (described below), and is only supported for compatibility with filter lists written for other ad blocking software.
```

Comment:
We need to mention that we're doing that for compatibility with Adblock Plus.

One thing that's missing is handling case which we also need to mention:
```
abptestpages.org##.testcase-inline-css-ehe {background-color: #c70d2c;}
abptestpages.org#@#.testcase-inline-css-ehe
```